csrc/scheduler/normalization_utils.h

```diff
@@ -322,7 +323,7 @@ bool isProjectBufferToInputs(
 
 // move persistent buffer marked in rparams->smem_persistent_buffers from
 // register to smem
-void movePersistentBufferToSmem(
+std::vector<TensorView*> movePersistentBufferToSmem(
```

Comment:
Please update the comment about the return value

Reply:
updated